specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2019-08-02/HybridCompute.json

```diff
@@ -0,0 +1,1255 @@
+                   "$ref":"#/parameters/ApiVersionParameter"
+                },
+                {
+                   "$ref":"#/parameters/SubscriptionIdParameter"
+                }
+             ],
+             "responses":{
+                "200":{
+                   "description":"OK",
+                   "schema":{
+                      "$ref":"#/definitions/MachineListResult"
+                   }
+                },
+                "default":{
+                   "description":"Error response describing why the operation failed.",
+                   "schema":{
+                      "$ref":"#/definitions/ErrorResponse"
+                   }
+                }
+             },
+             "x-ms-examples":{
+             },
+             "x-ms-pageable":{
+                "nextLinkName":"nextLink"
+             }
+          }
+       },
+       "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridCompute/machines/{name}/extensions/{extensionName}":{
+          "put":{
+             "tags":[
+                "MachineExtensions"
+             ],
+             "operationId":"MachineExtensions_CreateOrUpdate",
+             "description":"The operation to create or update the extension.",
+             "parameters":[
+                {
+                   "name":"resourceGroupName",
+                   "in":"path",
+                   "required":true,
+                   "type":"string",
+                   "description":"The name of the resource group."
+                },
+                {
+                   "name":"name",
+                   "in":"path",
+                   "required":true,
+                   "type":"string",
+                   "description":"The name of the machine where the extension should be created or updated."
+                },
+                {
+                   "name":"extensionName",
+                   "in":"path",
+                   "required":true,
+                   "type":"string",
+                   "description":"The name of the machine extension."
+                },
+                {
+                   "name":"extensionParameters",
+                   "in":"body",
+                   "required":true,
+                   "schema":{
+                      "$ref":"#/definitions/MachineExtension"
+                   },
+                   "description":"Parameters supplied to the Create Machine Extension operation."
+                },
+                {
+                   "$ref":"#/parameters/ApiVersionParameter"
+                },
+                {
+                   "$ref":"#/parameters/SubscriptionIdParameter"
+                }
+             ],
+             "responses":{
+                "200":{
+                   "description":"OK",
+                   "schema":{
+                      "$ref":"#/definitions/MachineExtension"
+                   }
+                },
+                "202":{
+                   "description":"Accepted"
```

Comment:
Can you add more text about the async operation status or how to poll?

Reply:
Updated.